DEMO/LEDsetup.py

```diff
@@ -1,21 +1,16 @@
-
-# coding: utf-8
-
 # In[1]:
 
-# The lines reset a driver called spidev, 
+# The lines reset a driver called spidev,
 # that is used to drive the signal sent to the lights from the raspberry pi
 from subprocess import call
 call(["sudo", "chmod", "a+rw", "/dev/spidev0.0"])
 
 # Importing some parts of a 'library' of programs called bibliopixel
-# Bibliopixel has the all the functions we use to light up LEDs 
+# Bibliopixel has the all the functions we use to light up LEDs
 # like led.set(), led.fill() etc
 
-from bibliopixel import *
```

Comment:
Please, please say you won't teach students to do `import *` for a library.

It's a shortcut that doesn't save very much typing, but it has bad consequences - [see this](https://stackoverflow.com/questions/2386714/why-is-import-bad) for more information.

Most modern codebases completely forbid `import *`.

Yes, I know our earlier code samples did this.  :-/  They don't any more!